.github/workflows/main.yml

```diff
@@ -106,7 +106,7 @@ jobs:
     needs: build
     strategy:
       matrix:
-        os: [ "ubuntu-22.04", "macos-13", "windows-2022" ]
+        os: [ "ubuntu-22.04", "macos-14", "windows-2022" ]
```

Comment:
Are the macos-14 agents ARM64 agents? Feels weird to me that doing a bump in OS version here would also change the underlying CPU architecture?

Reply:
[Apparently](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) (I know nothing about macOS!)